src/nav/navActions.js

```diff
@@ -1,12 +1,9 @@
 /* @flow */
 import { NavigationActions } from 'react-navigation';
 
-import type { NavigateAction, GetState, Message, Narrow, ApiServerSettings } from '../types';
-import { getSameRoutesCount } from '../selectors';
+import type { NavigateAction, Message, Narrow, ApiServerSettings } from '../types';
 
-export const navigateBack = () => (dispatch: Dispatch, getState: GetState): NavigateAction =>
-  // $FlowFixMe
-  dispatch(NavigationActions.pop({ n: getSameRoutesCount(getState()) }));
```

Comment:
The commit message says `pop` was removed, which made me concerned that the preceding commit was putting us in a broken state.

I think it isn't, though -- here it still is in the docs:
https://reactnavigation.org/docs/en/navigation-prop.html#pop

... Oh, subtle. This is Navigation*Actions*.pop, which has indeed gone missing from the docs.

Before checking the docs, I looked in the release notes.
https://reactnavigation.org/blog/2018/05/07/react-navigation-2.0.html
For a major-version upgrade like this, I wanted to be sure to read those anyway. I didn't see this change there, and it'd be pretty disappointing if they broke part of the API like that and didn't mention it... but aha, there's this bit:

> NavigationActions split up according to router
> If you are using NavigationActions.push or other stack-specific actions, you’ll need to import StackActions and use StackActions.push instead.

And indeed, here's that `pop` method:
https://reactnavigation.org/docs/en/stack-actions.html

So what I think I'd like to do is:
* We shouldn't leave things in a broken state at any commit. The commit that bumps the react-navigation version should fix this line so it keeps working, in the minimal way that's easy to see is correct: switch to `StackActions`.
* This switch to `back` looks cleaner than `getSameRoutesCount`, so I'll be glad to see a change that does that. It should be its own followup commit, and the commit message should explain why the new thing really does have the exact same behavior as the old -- that isn't obvious to me from a quick look at the doc for `back`. (Or it'd be fine for the behavior to be different; then the explanation just needs to say why the new behavior is good.)